src/main/resources/db/migration/V110__batch_court_changes_18.sql

```diff
@@ -0,0 +1,152 @@
+--- FACT-591 ---
+DELETE FROM public.search_courtareaoflaw
+WHERE area_of_law_id = 34254
```

Comment:
Is that better to use the name for 'money claims' rather than the auto-generated ID here? e.g.:

DELETE FROM public.search_courtareaoflaw
WHERE area_of_law_id = (SELECT id from public.search_areaoflaw where name = 'Money claims')